docs/src/manifolds/symplectic.md

```diff
@@ -0,0 +1,25 @@
+# Symplectic 
+
+The [`Symplectic`](@ref) manifold, denoted $\operatorname{Sp}(2n)$, represents transformations into symplectic subspaces which keep the
```

Comment:
```suggestion
The [`Symplectic`](@ref) manifold, denoted $\operatorname{Sp}(2n, \mathbb{F})$, is a submanifold of $\mathbb{F}^{2n \times 2n}$ that represents transformations into symplectic subspaces which keep the
```
I think it's better to mention the embedding earlier and possibly leave it open for the complex symplectic group even if it isn't implemented yet.